app/src/main/res/layout/fragment_speaker_detail.xml

```diff
@@ -57,10 +57,10 @@
             android:id="@+id/speaker_tag_line"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:layout_marginEnd="8dp"
-            android:layout_marginStart="8dp"
             android:layout_marginTop="8dp"
             android:paddingBottom="32dp"
+            android:paddingStart="8dp"
+            android:paddingEnd="8dp"
```

Comment:
Probably we can fix this by using layout_width="0dp". We can layout it by Constraint. 👍 What do you think?

```
   android:layout_width="0dp"
```

Reply:
Thank you for review and comment 😃 
I tried your solution, but please look Screenshot. 
I think you expected  text left alignment. It's OK?

<img src="https://user-images.githubusercontent.com/11015997/34994367-a2157d98-fb16-11e7-97c5-050822f8541d.png" width="300" />

Reply:
Thank you for checking! I think before is better 😄